dev-lang/perl/patches/perl-5.40.0.patchset

```diff
@@ -513,32 +492,29 @@ index 25ace13..8a04ead 100644
  __END__
  
 -- 
-2.21.0
+2.45.1
 
 
-From 2f7ece189875184caa314e5362a27044f54b80d7 Mon Sep 17 00:00:00 2001
+From 587853920e03a11121c9d26c51b63080256d1bb5 Mon Sep 17 00:00:00 2001
 From: Jerome Duval <jerome.duval@gmail.com>
 Date: Fri, 9 Jun 2017 21:30:33 +0200
 Subject: disable fstack-protector for Haiku
 
 
-diff --git a/Configure b/Configure
-index 684a9c0..f0b1d8a 100755
---- a/Configure
-+++ b/Configure
-@@ -5500,6 +5500,7 @@ default|recommended)
- 	# thing. (NWC)
- 	case "$osname" in
- 	amigaos) ;; # -fstack-protector builds but doesn't work
-+	haiku) ;; # -fstack-protector builds but doesn't work
- 	*)	case "$gccversion" in
- 		?*)	set stack-protector-strong -fstack-protector-strong
- 			eval $checkccflag
+diff --git a/hints/haiku.sh b/hints/haiku.sh
+index 0f09f53..b76c7c1 100644
+--- a/hints/haiku.sh
++++ b/hints/haiku.sh
+@@ -1 +1,3 @@
+-# haiku sets all its specifics via Configure
++# haiku sets nearly all its specifics via Configure
++
++ccflags="$ccflags -fno-stack-protector"
```

Comment:
This is now set via the hints file instead of directly in the Configure script because this was refactored upstream and this is probably the easiest way to do it now (see https://github.com/Perl/perl5/pull/17860).